src/ctelnet.cpp

```diff
@@ -1914,6 +1914,11 @@ void cTelnet::handle_socket_signal_readyRead()
         }
         else
         {
+            if( ch == TN_BELL )
```

Comment:
Ah, the "do it yourself" way... I'd rather have Mudlet work out of the box here.